BREAKINGCHANGES_dev.md

```diff
@@ -1,3 +1,8 @@
 1. The `jet_brains_toolbox` step was renamed to `jetbrains_toolbox`. If you're
    using the old name in your configuration file in the `disable` or `only`
    fields, simply change it to `jetbrains_toolbox`.
+
+2. The "dry run" command line argument has been replaced with the run mode
+   argument:
+    * `-n` => `-m dry`
+    * `--dry-run` => `--run-type dry`
```

Comment:
I personally do not agree with removing `--dry-run`. I would keep it as an alias for `--run-type dry` because
1. `--dry-run` is a standard name that people know and expect
2. There's not much benefit from removing it compared to the breakage it causes